### PR TITLE
Use SkyCoordinates to calculate source pos, do not calculate theta

### PR DESCRIPTION
**Pull request by MaxNoe**
This is better because:

* We do not depend on the source transformations in fact-tools, which are known to be imprecise compared to astropy
* The application of the model in `klaas_apply_disp_regressor` becomes independent of a known source position.

For the analysis of known source positions, theta can later be calculated. 
I will add a script to do that.

### CODE REVIEW

File: klaas/scripts/train_disp_regressor.py [hunk 1]
```diff
@@ -76,18 +84,21 @@ def main(configuration_path, signal_path, predictions_path, disp_model_path, sig
...
     )
-
     log.info('Total number of events: {}'.format(len(df)))
+
+    source_x, source_y = horizontal_to_camera(
+        az=df[az_source_col], zd=df[zd_source_col],
+        az_pointing=df[az_pointing_col], zd_pointing=df[zd_pointing_col],
+    )
+
     df['true_disp'] = euclidean_distance(
```
Comment (mackaiver) [inline klaas/scripts/train_disp_regressor.py]: I think I remember having different naming conventions for true and predicted values. Was it ´disp_true´ and ´disp_prediction´  or something?

Comment (MaxNoe) [reply klaas/scripts/train_disp_regressor.py]: Those are not saved, just internally used in this script

Comment (MaxNoe) [reply klaas/scripts/train_disp_regressor.py]: The values that are store are 

* `source_x_prediction`
* `source_y_prediction`
* `disp_prediction`

Which imho comply to our naming conventions

**Comment by mackaiver** [comment]
So when applying the disp regressor we could also write ra_prediction and dec_prediction to the output no?

**Comment by MaxNoe** [comment]
Only for data, but yes, we could.

**Comment by mackaiver** [comment]
I agree. We should have a timestamp in the montecarlos and in the data. Not this unix_time blabla

**Pull request merged**